src/main/java/mate/academy/lib/Injector.java

```diff
@@ -82,7 +82,7 @@ private Object getNewInstance(Class<?> certainClass) {
         return newInstance;
     }
```

Comment:
This method is private and static, but it's not used anywhere in the class. Consider removing it.